### PR TITLE
Expose RabbitMQ Prometheus metrics

### PR DESCRIPTION
**Pull request by jeandemeusy**

### CODE REVIEW

File: day-2/rabbitmq/application.yaml [hunk 1]
```diff
@@ -20,6 +20,10 @@ spec:
       releaseName: rabbitmq
       valueFiles:
         - ./values.yaml
+      service:
+        type: ClusterIP
+        ports:
+          http: 15692
```
Comment (ausias-armesto) [inline day-2/rabbitmq/application.yaml:26]: Helm value changes should be placed in the dedicated file https://github.com/hoprnet/ct-platform/blob/main/day-2-definitions/rabbitmq/values.yaml

File: day-2/rabbitmq/service-monitor.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```
Comment (ausias-armesto) [inline day-2/rabbitmq/service-monitor.yaml:2]: According to the helm chart, the monitoring is already included. Only needs to be enabled

https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq-cluster-operator/values.yaml#L457

**Review by ausias-armesto — CHANGES_REQUESTED** [review]

File: day-2/rabbitmq/application.yaml (hunk 1 above)
Comment (ausias-armesto) [reply day-2/rabbitmq/application.yaml:26]: Why do you need to expose that port?
It should be done if you use the feature of enabling the monitoring

Comment (jeandemeusy) [reply day-2/rabbitmq/application.yaml:26]: I saw that this is the port on which prometheus metrics are exposed

File: day-2/rabbitmq/service-monitor.yaml (hunk 2 above)
Comment (jeandemeusy) [reply day-2/rabbitmq/service-monitor.yaml:2]: According to the `values.yaml`, monitoring is already enabled. However I cannot any metric related to RabbitMQ on prometheus

**Review by ausias-armesto — APPROVED** [review]

**Pull request merged**